portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,25 +38,33 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
-    PreparedQuery results = datastore.prepare(new Query("Comment"));
-    List<String> comments = new ArrayList<>();
+    PreparedQuery results =
+        datastore.prepare(new Query("Comment").addSort("timestamp", SortDirection.DESCENDING));
+    List<Comment> comments = new ArrayList<>();
     for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
+      String name = (String) entity.getProperty("name");
```

Comment:
make property "name" and others to be private static final String, and use the constants in other places as well.

Reply:
done